lib/mpool/lib/mblock_file.c

```diff
@@ -583,14 +576,22 @@ mblock_file_meta_log(struct mblock_file *mbfp, uint64_t *mbidv, int mbidc, bool
  */
 
 void
-mblock_wlen_set(struct mblock_file *mbfp, uint64_t mbid, uint32_t wlen, bool prealloc)
+mblock_wlen_set(
```

Comment:
Can this be static?  Seems like setting `wlen` should not be a widely available function.

Reply:
I had to expose this function so that the mblock clone implementation in mblock_fset.c can use it to set the write length of the target mblock after the clone operation.